ska_builder.py

```diff
@@ -137,6 +180,11 @@ def build_one_package(name, tag=None):
     build_package(name)
     print('')
 
+    if args.github_org:
+        print('Removing git clone because --github-org was supplied')
+        clone_path = os.path.join(SRC_DIR, name)
+        shutil.rmtree(clone_path)
+
```

Comment:
This is another thing... you are removing the directory if you want to force the build? This does more than forcing the build. The --force used to just make sure conda didn't skip it. This starts fresh.

I did not like much the previous way, where the script could be fooled into building something with extra commits in the local index, and I though we should check for that if we wanted to ensure a clean build.

This ensures the clean build, but only if you force the conda build.

Reply:
This is removing the repo in `src` only when you supply `--github-org` (which is rare). Think about `backstop_history` that comes from `acisops`.  If you have been building that before and have repo in your `src` dir cloned from `acisops`, and then supply `--github-org=sot` then it won't work because `origin` is acisops not sot.  So I decided to be safe to just disable the git repo caching in `src` when you are playing around with the org with `--github-org`.

I don't quite understand your question related to `--force` since that is not related to this code. Even if there is no repo it won't "force" the build if the version is the same, though of course without a repo it will always re-clone the repo.

Reply:
ooops, right. I don't know why I read --force.